.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,180 @@
+# ./.github/workflows/staging-auto-pr.yaml
+name: create draft release
+on:
+  workflow_dispatch:
+    inputs:
+      beta_version:
+        required: false
+        default: false
+        description: beta version
+      publish_release:
+        required: false
+        default: false
+        description: publish release
+
+env:
+  GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL_TEST }}
+
+jobs:
+  run-regressions:
+    name: run regressions tesing
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Run full regressions tests"
+        run: "echo full regressions tests...."
+
+  update-readme:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    needs: run-regressions
+    name: update weka version
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: dev
+          fetch-depth: 0
+
+      - name: Bump version and push tag
+        uses: anothrNick/github-tag-action@1.64.0 # Don't use @master or @v1 unless you're happy to test the latest version
+        id: tag_version
+        env:
+          WITH_V: false
+          PRERELEASE: ${{ inputs.beta_version }}
```

Comment:
why do we need it?

Reply:
not sure, but maybe we will need to create a beta version for qa